src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml

```diff
@@ -9,6 +9,8 @@
 	<![CDATA[
 		Issue 2538: Strict-Transport-Security Header Scanner additional variants.<br>
 		Issue 2716: Implement a passive check for cookies without SameSite set.<br>
```

Comment:
Yup sorry for the confusion, when I mentioned 11 I meant v11 of the addon. So yes both lines 10 and 11 can be removed.

<sub> (https://github.com/zaproxy/zap-extensions/blob/3f9572289db0eb3a78a6d7557a73ddeb2f786d7e/src/org/zaproxy/zap/extension/pscanrulesAlpha/ZapAddOn.xml#L10-L11)</sub>